android/demo/src/com/mapzen/tangram/android/MainActivity.java

```diff
@@ -170,5 +213,52 @@ public void onFeaturePick(Map<String, String> properties, float positionX, float
                 "Selected: " + name + " at: " + positionX + ", " + positionY,
                 Toast.LENGTH_SHORT).show();
     }
-}
 
+    private void saveScreenshot(Bitmap bitmap) {
+
+        ByteArrayOutputStream bos = new ByteArrayOutputStream();
+        bitmap.compress(Bitmap.CompressFormat.PNG, 90, bos);
```

Comment:
I think you can pass a `FileOutputStream` directly to `Bitmap$compress` and skip the byte array altogether.